OpenRA.Game/Player.cs

```diff
@@ -130,20 +130,27 @@ public Player(World world, Session.Client client, PlayerReference pr)
 			fogVisibilities = PlayerActor.TraitsImplementing<IFogVisibilityModifier>().ToArray();
 
 			// Enable the bot logic on the host
-			IsBot = botType != null;
-			if (IsBot && Game.IsHost)
+			ActivateBot(botType);
+
+			stanceColors.Self = ChromeMetrics.Get<Color>("PlayerStanceColorSelf");
+			stanceColors.Allies = ChromeMetrics.Get<Color>("PlayerStanceColorAllies");
+			stanceColors.Enemies = ChromeMetrics.Get<Color>("PlayerStanceColorEnemies");
+			stanceColors.Neutrals = ChromeMetrics.Get<Color>("PlayerStanceColorNeutrals");
+		}
+
+		public void ActivateBot(string botType)
+		{
+			if (!string.IsNullOrEmpty(botType) && Game.IsHost)
 			{
 				var logic = PlayerActor.TraitsImplementing<IBot>().FirstOrDefault(b => b.Info.Name == botType);
 				if (logic == null)
 					Log.Write("debug", "Invalid bot type: {0}", botType);
 				else
+				{
 					logic.Activate(this);
+					Log.Write("debug", "Activated bot {0}".F(botType));
```

Comment:
This causes the player chat to be spammed at the start of the game.  We've had bots for years, and nobody has ever said that they wanted or needed to know this (since it has always been obvious).

Players are already informed by other log messages in the admin-drop case.